json-rpc/src/tests/unit_tests.rs

```diff
@@ -393,6 +393,26 @@ fn test_get_transactions_impl(blocks: Vec<(Vec<TransactionToCommit>, LedgerInfoW
     }
 }
 
+#[test]
+fn test_get_state_proof() {
+    let address = format!("0.0.0.0:{}", utils::get_available_port());
+    let _runtime = bootstrap(
+        address.parse().unwrap(),
+        Arc::new(mock_db(vec![])),
+        channel(1024).0,
+    );
+    let client = reqwest::blocking::Client::new();
+    let url = format!("http://{}", address);
+
+    let version = 10;
```

Comment:
how did you get 10?

Reply:
mock will just respond whatever you give it to it